GameData/RealismOverhaul/Engine_Configs/TinyTim_Config.cfg

```diff
@@ -166,43 +179,70 @@
+				isSolid = True
+				techTransfer = 30Klbf:80
+			}
 		}
 	}
+
 }
 
-@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[30Klbf]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]:NEEDS[TestLite|TestFlight]
+// TODO: move elsewhere. probably wherever FOR[zTestFlight] lives.
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]]:BEFORE[zTestFlight]
 {
-	TESTFLIGHT
-	{
-		name = 30Klbf
-		ratedBurnTime = 5	//account for thrust tail-off
-		ignitionReliabilityStart = 0.98 // starts higher than expected
-		ignitionReliabilityEnd = 0.9999 // literal steel tube of explosives, thousands built and used in pacific theatre
-		ignitionDynPresFailMultiplier = 50	//started life as anti-ship rocket
-		// Because the fail chance is 10x during the first 5 seconds of burn, this needs to be 10x as reliable as you'd think.
-		cycleReliabilityStart = 0.999
-		cycleReliabilityEnd = 0.9997
-		reliabilityDataRateMultiplier = 1.5
-		isSolid = True
-		techTransfer = 50Klbf:80
+	@MODULE[ModuleEngineConfigs] {
+		@CONFIG:HAS[@TESTFLIGHT] {
+			|_ = CONFIG_wip
+			@TESTFLIGHT,* {
+				name = #$../name$
+			}
+		}
 	}
-	@MODULE[ModuleEngineConfigs] { @CONFIG[30Klbf] { %ratedBurnTime = #$/TESTFLIGHT[30Klbf]/ratedBurnTime$ } }
 }
-@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[50Klbf]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]:NEEDS[TestLite|TestFlight]
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG_wip]]:BEFORE[zTestFlight]
 {
-	TESTFLIGHT
+	// Copy the TESTFLIGHT where FOR[zTestFlight] expects it; but before
+	// deleting the CONFIG-nested original, we'll use i to populate some
+	// CONFIG fields.
+	#/MODULE[ModuleEngineConfigs]/CONFIG_wip/TESTFLIGHT {}
+
+	@MODULE[ModuleEngineConfigs]
 	{
-		name = 50Klbf
-		ratedBurnTime = 3	//account for thrust tail-off
-		ignitionReliabilityStart = 0.98 // starts higher than expected
-		ignitionReliabilityEnd = 0.9999 // literal steel tube of explosives, thousands built and used in pacific theatre
-		ignitionDynPresFailMultiplier = 50	//started life as anti-ship rocket
-		// Because the fail chance is 10x during the first 5 seconds of burn, this needs to be 10x as reliable as you'd think.
-		cycleReliabilityStart = 0.999 // starts higher than expected
-		cycleReliabilityEnd = 0.9997
-		reliabilityDataRateMultiplier = 1.5
-		isSolid = True
-		techTransfer = 30Klbf:80
+		@CONFIG_wip
+		{
+			@TESTFLIGHT {
+				&ratedBurnTime = dummy
+				&ratedContinuousBurnTime = dummy
+				&testedBurnTime = dummy
+			}
+		}
+		@CONFIG_wip
+		{
+			%ratedBurnTime = #$TESTFLIGHT/ratedBurnTime$
+			%ratedContinuousBurnTime = #$TESTFLIGHT/ratedContinuousBurnTime$
+			// TODO: maybe something like this to display testedBurnTime SOMEWHERE in-game
+			// @description = #$description$ testedBurnTime=$TESTFLIGHT/testedBurnTime$
+			// @description ^= : testedBurnTime=dummy$::
+		}
+		@CONFIG_wip:HAS[#ratedBurnTime[dummy]] { !ratedBurnTime = delete }
+		@CONFIG_wip:HAS[#ratedContinuousBurnTime[dummy]] { !ratedContinuousBurnTime = delete }
```

Comment:
Maybe

```
@CONFIG_wip:HAS[@TESTFLIGHT:HAS[#ratedBurnTime]] { &ratedBurnTime = #$TESTFLIGHT/ratedBurnTime$ }
// &c.
```

Reply:
I think I wanted to avoid needing multiple nested lookups, but I didn't end up in a good place. Will use this.
(and then maybe later when it's all done, see if there's a perf difference)